src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { FC, useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import up from '../../asset/icon/expanded-up.svg';
+import down from '../../asset/icon/expanded-down.svg';
+import { classNames } from '../../utils/classNames';
+export type AccordionType = 'default' | 'checkbox';
```

Comment:
If you don't actually need this `AccordionType` type anywhere else, you could simply write it inside the `IAccordionProps` interface